### ISSUE TITLE
Support for web tag in comicbox.yaml scheme

### ISSUE THREAD

**Issue by warthan07** [ajslater/comicbox#73]
I was wondering whether the "web" tag could be supported in the comicbox.yaml scheme (to be further exported to other compatible schemes such as CIX). Looking at the following commented-out line, I think such a support was at least planned:

https://github.com/ajslater/comicbox/blob/5a428f9549fdee932d0ee430ca045ea3bcaa4493/comicbox/transforms/comicinfo.py#L103

On my local install, I successfully tested that adding the field `web = StringField()` to `ComicboxSubSchemaMixin` and importing correctly WEB_KEY in the cited file above allows a correct export of the web tag from comicbox cli to ComicInfo.xml. But I am not so sure about the compatibility with other schemes that I am not using, maybe there is a reason this tag was commented out?

**Comment by ajslater** [comment]
This was a little unintuitive, baroque and also broken.
Tagging Web with arbitrary urls should now be fixed in [v1.1.5](https://github.com/ajslater/comicbox/releases/tag/v1.1.5)
However it's still undeniably quirky and I've included a bit in the README because of this.

Basically URLs are gonna show up under the `identifiers.url` tag in comicbox.yaml but that shouldn't be a huge concern because the CLI metadata parser accepts heterogenous tag formats in yaml form with the `-m` option, so you can just use ComicInfo.xml's Web tag and it should work.
so like:
```sh
comicbox -p -m "Web: https://foo.com" mycomic.cbz
```

and the identifier tag should appear in comicbox.yaml as:

```yaml
identifiers:
  nss: foo.com
  url: https://foo.com
```

**Comment by warthan07** [comment]
Hoo, I looked at your commits and I understand better why my local hack was not good :) You are perfectly right, my url was not recognised because it is a bedetheque.com url (I mostly read francophone "bandes dessinées") that is not matched by the WEB_REGEX_URLS. Thanks in any case for pushing this so quickly :) 

As a side note, it would likely be possible to parse the unique comic identifier (the NSS_KEY if I understood correctly) from bedetheque.com urls, but at least for me it is not very useful because this information is not used in the CIX scheme.
